Transbank/WebpayRest/TransaccionCompleta/Transaction.cs

```diff
@@ -0,0 +1,81 @@
+using System;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Serialization;
+using Transbank.Common;
+using Transbank.Exceptions;
+using Transbank.Webpay.Common;
+using Transbank.Webpay.Oneclick.Requests;
+using Transbank.Webpay.TransaccionCompleta.Requests;
+using Transbank.Webpay.TransaccionCompleta.Responses;
+
+namespace Transbank.Webpay.TransaccionCompleta
```

Comment:
This should be named `FullTransaction` instead of `TransaccionCompleta`

Reply:
Fixed as you say at 15678a2